examples/providerconfig/provider.yaml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
It is good practice too keep the YAML document separator at the start of a YAML file even though it only contains a single document.